### PR TITLE
htlc basic integration test infrastructure

### PR DESCRIPTION
**Pull request by HagarMeir**
Signed-off-by: Hagar Meir <hagar.meir@ibm.com>

### CODE REVIEW

**Comment by HagarMeir** [comment]
https://github.com/hyperledger-labs/fabric-token-sdk/issues/255

File: integration/token/interop/fabtoken/fabtoken_test.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import (
+	"runtime"
+
+	"github.com/hyperledger-labs/fabric-smart-client/integration"
+	"github.com/hyperledger-labs/fabric-token-sdk/integration/nwo/token"
+	"github.com/hyperledger-labs/fabric-token-sdk/integration/token/interop"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	integration2 "github.com/hyperledger-labs/fabric-token-sdk/integration"
```
[the remark this answers is not in the document]
Comment (mbrandenburger) [reply integration/token/interop/fabtoken/fabtoken_test.go:18]: please check the imports for other files as well :)

**Review by mbrandenburger — COMMENTED** [review]
LGTM - just a few minor comments.

**Comment by HagarMeir** [comment]
closing this and continuing with [https://github.com/hyperledger-labs/fabric-token-sdk/pull/263](url)

File: .github/workflows/interop_fabtoken.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+name: Interop with FabToken Driver
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main, htlc ]
```
[the remark this answers is not in the document]
Comment (HagarMeir) [reply .github/workflows/interop_fabtoken.yml:7]: true

File: integration/token/interop/fabtoken/fabtoken_test.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+		ii *integration.Infrastructure
+	)
+
+	BeforeEach(func() {
+		token.Drivers = append(token.Drivers, "fabtoken")
+	})
+
+	AfterEach(func() {
+		ii.Stop()
+	})
```
[the remark this answers is not in the document]
Comment (HagarMeir) [reply integration/token/interop/fabtoken/fabtoken_test.go:32]: we could try this with other integration tests first

File: integration/token/interop/fabtoken/fabtoken_test.go [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+		})
+	})
+
+	Describe("Asset Exchange Two Fabric Networks", func() {
+		BeforeEach(func() {
+			var err error
+			testDir := ""
+			if runtime.GOOS == "darwin" {
+				testDir = "./testdata"
+			}
```
[the remark this answers is not in the document]
Comment (HagarMeir) [reply integration/token/interop/fabtoken/fabtoken_test.go:59]: not sure
removed it

**Pull request closed without merging**